README.md

```diff
@@ -32,11 +32,11 @@ Save the therapist email address and password in your `.env` file as THERAPIST_E
 See .env-example for a template.
 
 ## Run application
-Run `script/server` script to serve the application, or follow these steps:
-1. Run `script/bootstrap` script
-2. Start everything with `docker compose up`.
+Run `script/server` script to serve the application.
 
-NB: If you get a problem about "subdir not supported" during `docker-compose up`, set the buildkit feature to false in Docker. On Windows, you can do this in Docker Desktop>Settings>Build engine.
+NB: If you get a problem about "subdir not supported" during execution of `script/server`, 
+set the buildkit feature to false in Docker. 
+On Windows, you can do this in Docker Desktop>Settings>Build engine.
 
```

Comment:
```suggestion
On Mac, you can do this in Docker Desktop>Preferences>Docker Engine. Edit the displayed JSON so that `"buildkit": false`, then restart Docker Desktop.
```